qa/src/test/resources/io/camunda/migrator/bpmn/c8/userTaskProcess.bpmn

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?><bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:conversion="http://camunda.org/schema/conversion/1.0" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" conversion:converterVersion="0.12.4" exporter="Camunda Modeler" exporterVersion="5.24.0" expressionLanguage="http://www.w3.org/1999/XPath" id="Definitions_1a10piy" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.6.0" targetNamespace="http://bpmn.io/schema/bpmn" typeLanguage="http://www.w3.org/2001/XMLSchema">
+  <bpmn:process id="userTaskProcessId" isClosed="false" isExecutable="true" name="UserTaskProcess" processType="None">
+    <bpmn:extensionElements>
+      <conversion:message severity="INFO">Unused attribute 'historyTimeToLive' on 'process' is removed.</conversion:message>
+    </bpmn:extensionElements>
+    <bpmn:startEvent id="StartEvent_1" isInterrupting="true" name="Start" parallelMultiple="false">
+      <bpmn:extensionElements>
+        <zeebe:executionListeners>
+          <zeebe:executionListener eventType="end" type="migrator" />
+        </zeebe:executionListeners>
+      </bpmn:extensionElements>
+      <bpmn:outgoing>Flow_0xg7od7</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:sequenceFlow id="Flow_0xg7od7" sourceRef="StartEvent_1" targetRef="userTaskId"/>
+    <bpmn:endEvent id="Event_0togu4y" name="End">
+      <bpmn:incoming>Flow_1ur86ut</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1ur86ut" sourceRef="userTaskId" targetRef="Event_0togu4y"/>
+    <bpmn:userTask completionQuantity="1" id="userTaskId" implementation="##unspecified" isForCompensation="false" name="UserTaskName" startQuantity="1">
```

Comment:
❓  Based on [docs](https://docs.camunda.io/docs/components/modeler/bpmn/undefined-tasks/) `implementation="##unspecified"` means the task is a pass through / NoOp. 
Would it make sense to have a test with embedded and external forms?

Reply:
Hm I dont know what that means, I'll read up on the docs

Reply:
Ok very odd. The docs sound like the instance would move through the element but in our test it waits at the user task (which is what I expected for user tasks) 🤔

Reply:
Glean says the following:
```
 Typically, in broader BPMN usage, this could indicate a pass-through or NoOp behavior. However, in Camunda 8, user tasks with implementation="##unspecified" are treated as regular user tasks. They create a wait state and pause execution until completed by a user, behaving just like any other user task. This attribute is likely a remnant from broader BPMN usage or from Camunda 7, where it sometimes indicated special behavior but has no effect in Camunda 8 models
 ```
 So I think its fine. Wdyt?